src/Tests/ConfigUpdaterTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Drupal\commerce\Tests;
+
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Tests the ConfigUpdater class.
+ *
+ * @group commerce
+ */
+class ConfigUpdaterTest extends WebTestBase {
```

Comment:
This test doesn't cover all of the failure modes (trying to import a non-existent config name, etc).